R/tsplot.R

```diff
@@ -147,8 +147,26 @@ tsplot.list <- function(...,
   cnames <- names(tsl)
   # if(!is.null(tsr)) cnames <- names(tsr) 
   
-  tsl_r <- range(as.numeric(unlist(tsl)),na.rm = T)
-  tsl_r_true <- tsl_r
+  if(left_as_bar) {
+    # Combine ts
+    tsmat <- do.call("ts.union", tsl)
```

Comment:
does a call to ts.union work for all types of time series that we use: ts, xts, zoo?

Reply:
1) No, we'll need to use cbind for that
2) For non-ts objects, fillUpYearWithNAs throws a warning on line 12
```
Error in `-.yearqtr`(frq, end(x)[2]) : 
  Can only subtract from yearqtr objects
```